scripts/bulk-password-update-export.js

```diff
@@ -67,6 +69,25 @@ Do you want to continue? [y/N]
   }));
 };
 
+const changeUserPass = async (user, password, options) => {
+  const admins = ['admin', 'medic'];
```

Comment:
I would pull this out as a top level constant. There's no need to init the same array over and over again!